src/test/java/com/brewmes/demo/BrewMESApplicationTests.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
-class BrewMESApplicationTests {
+class BrewMesApplicationTests {
 
 	@Test
 	void contextLoads() {
```

Comment:
Why does this class even exist?

Reply:
Its a default testclass made by SpringBoot. After the changing of names this class caused me troubles so i had to rename it. Not sure if we can just delete it.